java/java.lsp.server/vscode/src/test/suite/extension.test.ts

```diff
@@ -34,7 +36,7 @@ suite('Extension Test Suite', () => {
 
     test('Find clusters', async () => {
         let clusters = myExtension.findClusters('non-existent');
-        assert.strictEqual(clusters.length, 6, 'six clusters found: ' + clusters);
+        assert.strictEqual(clusters.length, 7, 'seven clusters found: ' + clusters);
```

Comment:
This can certainly be changed by knowing the minimal list of clusters to find and ignoring others.